### ISSUE TITLE
Create CNN network

### ISSUE THREAD

**Issue by ShkalikovOleh** [ShkalikovOleh/NExp#10]
Create convolutional neural network model. This model contains:

- convolutional layer
- subsampling layer
- softmax layer
